File: app/src/main/java/com/gu/swiperefreshplush/MainActivity.java
package com.gu.swiperefreshplush;

import android.app.Fragment;
import android.support.design.widget.TabLayout;
import android.support.v4.view.ViewPager;
import android.support.v4.widget.SwipeRefreshLayout;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.support.v7.widget.Toolbar;
import android.view.View;

import com.gu.swiperefreshplush.fragment.DataPresenter;
import com.gu.swiperefreshplush.fragment.ListFragment;
import com.gu.swiperefreshplush.fragment.RecycleFragment;

public class MainActivity extends AppCompatActivity {
    private ViewPager mContent;
    private TabLayout mTabLayout;
    private Toolbar mToolbar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        mContent= (ViewPager) findViewById(R.id.vp_content);
        mTabLayout=(TabLayout) findViewById(R.id.tab_layout);
        mToolbar=(Toolbar)findViewById(R.id.tool_bar);
        setSupportActionBar(mToolbar);
        mContent.setAdapter(new PagerAdapter(getSupportFragmentManager()));
        mTabLayout.setupWithViewPager(mContent);
    }

}

File: app/src/main/java/com/gu/swiperefreshplus/MainActivity.kt
package com.gu.swiperefreshplus

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setSupportActionBar(tool_bar)
        vp_content.adapter = PagerAdapter(supportFragmentManager)
        tab_layout.setupWithViewPager(vp_content)
    }

}
